PokeDemo/CustomViews/TableViewCells/PokemonListCell.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  PokemonListCell.swift
+//  PokeDemo
+//
+//  Created by Rudr Bansal on 05/07/22.
+//
+
+import UIKit
+
+final class PokemonListCell: UITableViewCell {
+    
+    // MARK: - Outlets
+    
+    @IBOutlet weak var title: UILabel!
```

Comment:
weak not required

Reply:
Covered in c141e033217fe7cb155df5baff1dbc9b44c0b899